PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -56,13 +70,15 @@ class PayPalWebCheckoutClient internal constructor(
      *
      * @param request [PayPalWebCheckoutRequest] for requesting an order approval
      */
-    fun start(
+    suspend fun start(
```

Comment:
```suggestion
    private suspend fun start(
```
It makes more sense to fully support coroutines in the next major version.

Reply:
Actually this is a breaking change. This method should be deprecated instead of converted to a `suspend` function.

Reply:
I'm surprised the CI tool didn't pick this up.

Reply:
it did flagged updated API. Thought we will go with major
deprecated this and vault functions